samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/delivery/SubmitToGradingActionListener.java

```diff
@@ -99,6 +101,7 @@
 
 	private final PreferencesService preferencesService = ComponentManager.get( PreferencesService.class );
 	private final UserDirectoryService userDirectoryService = ComponentManager.get( UserDirectoryService.class );
+	private String studentRichText = ServerConfigurationService.getString("samigo.studentRichText", "true");
```

Comment:
Please use ComponentManager.get(ServerConfigurationService.class) first

and this should be a boolean

Reply:
Done!